paasta_itests/static_port.feature

```diff
@@ -1,25 +1,29 @@
 Feature: setting host_port in marathon.yaml creates a marathon app that behaves as we expect.
+  @skip
```

Comment:
Yep a bunch seem to behave differently locally vs travis. Probably they worked locally when you shipped @EvanKrall but never passed on travis because of the sys.exit thing above we never noticed...